.github/actions/latest-semver-tag/action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Latest Semantic Version Tag'
+description: 'Determines the latest tagged git release following semantic versioning'
+inputs:
+  release:
+    description: 'The release type to filter tags by'
+    required: true
+    default: 'main'
+    # Allowed values: main, fips1, fips2, fips3
+  path:
+    description: 'The path containing the git repository to check'
+    required: false
+    default: '.'
```

Comment:
Are there any examples of circumstances where this would come in handy? It seems like we'll be running the checks against this repo's root directory in most cases.

Reply:
This path is mainly for the instance where an alternative checkout path is provided to `actions/checkout`, which is the case for the ABI workflows since the repo is checked out twice (previous & next). But normally it would just default to current path `.` since thats also what `actions/checkout` does.